connectors/us/us-trust-agreement.md

```diff
@@ -1,44 +1,45 @@
-11. **Grantor** is the individual or entity who creates the Trust and grants the Property to the Trust.
-12. **Mint** means a Confirmed Transaction `mint()` or `mintBatch()` function on the Fabrica Smart Contract that results in the association of a Token ID to an Address.
-13. **Non Fungible Token (NFT)** means the digital asset stored on Ethereum with a unique identification code (Token ID) adhering to the ERC-1155 standards.
-14. **NFT Metadata** means the data directly stored within the NFT as well as external data stored on IPFS and linked within the NFT itself (using the fields `definition` and `configuration`). These will always include Token ID, Trust Name (stored as `definition.holdingEntity`), and Property legal description (`definition.claim`). Other additional information may be included to simplify property identification, verify past ownership and other activities.
-15. **Property** is the bundle of rights identified in the legal description stored in the NFT Metadata under the field `definition.claim`. The Property so described is the bundle of rights which is to be deeded into the Trust and held throughout the life of the Trust.
-16. **Smart Contract** means the bytecode deployed on a specific Ethereum Address which acts as a program to execute and run a series of processes or interactions.
-17. **Token ID** means the unique and immutable identifier determined on Trust creation and assigned to the corresponding Fabrica NFT. The Token ID is created based on the digital signature of mutiple fields combined, including this Agreement.
-18. **Transfer** means any operation performed through the Smart Contract that assigns the NFT to a new non null Address.
-19. **Trustee** means the individual or entity appointed by the Beneficiary as the Trustee of the Trust.
-20. **Trust Name** means the name of the Trust, more particularly defined in the NFT Metadata (in `definition.holdingEntity`).
+2. **Address** means a public key address on a Blockchain Network.
+3. **Beneficiary** means the individual or entity that is in control of the most recent Account owning the Fabrica NFT. In the event the Fabrica NFT has not yet been minted, or has been minted but has never yet been owned by any Account, the Beneficiary shall be the original Grantor. In the event ownership of the Fabrica NFT is fractionalized (as discussed further in **Sections 6.1. and 7.3.**), then the defined term Beneficiary as used in this Agreement shall apply to the various owners of the Fabrica NFT as a group.
+4. **Blockchain Network** means the blockchain network and the consensus blockchain for such network used to maintain records of ownership and management of the Fabrica NFT to which this Agreement is attached. While the Blockchain Network is initially assumed to be Ethereum, the Blockchain Network may change due to actions of the Beneficiary, including the wrapping of the NFT to a different blockchain network or the minting of the NFT on a different blockchain network.
```

Comment:
ok - just as an fyi my thinking here was to give the agreement a default in the event that some terms later in the agreement fit specifically to Ethereum. 

i've reviewed the agreement and think i've specified the few areas where this seems relevant, so I can just incorporate disclaimers about alternative blockchains in those areas in the new pr.

Reply:
meanwhile I'm leaning more and more toward migrating all Fabrica properties to Base, so Ethereum might not be the best "default" case. 
Regardless we want this agreement to be as generic as possible so that other projects can use it on any blockchains of their choice, so we shouldn't imply any default network.